api/v1beta1/aerospikecluster_validating_webhook.go

```diff
@@ -1607,3 +1611,15 @@ func ValidateAerospikeObjectMeta(aerospikeObjectMeta *AerospikeObjectMeta) error
 	}
 	return nil
 }
+
+func validateDNS(dnsPolicy v1.DNSPolicy, dnsConfig *v1.PodDNSConfig) error {
+	if dnsPolicy == v1.DNSDefault {
```

Comment:
`Default` policy is not going to work out of the box for us as we are dependent on cluster internal DNS lookup.
It might be possible only if coreDNS related configuration is set across all nodes, which is very unlikely.

Reply:
`Default` policy is not going to work out of the box for us as we are dependent on cluster internal DNS lookup.
It might be possible only if coreDNS related configuration is set across all nodes, which is very unlikely.